packages/main/src/plugin/container-registry.spec.ts

```diff
@@ -3994,7 +3994,7 @@ describe('createContainerLibPod', () => {
     const expectedOptions: PodmanContainerCreateOptions = {
       name: options.name,
       command: options.Cmd,
-      entrypoint: options.Entrypoint,
+      entrypoint: [options.Entrypoint as string],
```

Comment:
issue: we're missing some unit tests to check the multiple cases (single entry, no entry or array)

Reply:
Thanks for taking a look, I can add some additonal tests. Will try to get to that tomorrow.

Reply:
@benoitf added additional tests and rebased.